source/wp-content/themes/wporg-news-2021/sass/components/_local-header.scss

```diff
@@ -133,104 +128,87 @@
 						transform: rotate(180deg);
 					}
 
+					// Display the scrim when toggled.
 					&::before {
-						content: "";
-						display: block;
-						position: fixed;
-						top: 0;
-						left: 0;
-						width: 100%;
-						height: 100vh;
-						background-color: rgba(0, 0, 0, 0.5);
-						z-index: -1;
-						animation-duration: 400ms;
-						animation-name: overlay-fade-in;
+						opacity: 1;
 					}
 				}
 
 				~ .wp-block-categories {
 					display: flex;
-					opacity: 1;
-					animation-duration: 400ms;
-					animation-name: categories-fade-in;
-
-					@at-root {
-
-						@keyframes categories-fade-in {
-							from {
-								opacity: 0;
-							}
-
-							to {
-								opacity: 1;
-							}
-						}
-					}
 				}
 			}
-
-			&:focus {
-				~ label span {
-					outline: 1px dotted currentColor;
-				}
-
-			}
 		}
 
+		// The list of categories which is toggled
+		// on and off using the label+checkbox above.
 		.wp-block-categories {
+			list-style: none;
+			padding: 20px 0 50px 0;
 			display: none;
-			opacity: 0;
-			position: relative;
-			right: calc(var(--wp--custom--alignment--edge-spacing) * -1);
-			flex-direction: column;
+			flex-direction: row; // should change on mobile
 			flex-wrap: wrap;
-			width: calc(min(100%, var(--wp--custom--layout--wide-size)));
-			color: var(--bar-text-color);
-			list-style: none;
-			padding: 0 0 2em 0;
-			text-align: right;
-
-			@include break-mobile {
-				padding: 0 var(--wp--custom--alignment--edge-spacing);
-				position: absolute;
-				left: 0;
-				right: auto;
-				top: 75px;
-				max-height: calc(1rem * var(--wp--custom--body--typography--line-height) * var(--flex-items-per-column));
-				text-align: left;
-			}
 
+			// Override the default margin applied to the block.
 			[class*="wp-container-"] & {
-				margin-top: 0;
+				margin: 0;
 			}
 
 			li {
-				padding: 0.25em calc(var(--wp--custom--alignment--edge-spacing) - 4px) 0.25em 0;
-				border-right: 4px solid transparent;
+				width: 20%;
 
-				@include break-mobile {
-					padding: 0.25em 0;
+				a {
+					display: block;
 				}
 			}
 
+			// If currently viewing a category, highlighted
+			// that category in the list.
 			.current-cat {
 				font-weight: 700;
-				border-right-color: currentColor;
-
-				@include break-mobile {
-					border: none;
-
-					&::before {
-						content: "\0020\00B7\0020";
-					}
+				position: relative;
+
+				// On small viewports the current category is highlighted
+				// using a right-side border.
```

Comment:
is this still in-progress?

Reply:
Oh my. I somehow had changes on my local that weren't pushed to this PR. Sorry about that. Many of these "in-progress" things have been fixed.